examples/token-classification/run_tf_ner.py

```diff
@@ -277,5 +281,10 @@ def compute_metrics(p: EvalPrediction) -> Dict:
     return results
 
 
+def _mp_fn(index):
+    # For xla_spawn (TPUs)
+    main()
```

Comment:
`xla_spawn.py` is for pytorch/xla, it shouldn't work with a TF script.

Reply:
Ok then can be claused.